assets/email_template.txt

```diff
@@ -1,6 +1,17 @@
-=================================================
- nf-core/methylseq: Bisulfite-Seq Best Practice v${version}
-=================================================
+=======
+========================================
+ nf-core/methylseq v${version}
+========================================
+=======
```

Comment:
One minor thing I noticed:
lines 1-5 should be deleted.

Reply:
Good spot - added..